### DISCUSSION TITLE
Is it possible to modify the arrow when using rdkit.Chem.Draw.ReactionToImage() ?

### DISCUSSION THREAD

**Discussion by agrodet** [rdkit/rdkit#6383] [Q&A] [answered]
Hello. Suppose I have the following code 

```
from rdkit.Chem import Draw, AllChem

rxn = AllChem.ReactionFromSmarts(template)
img = Draw.ReactionToImage(rxn)
```
img is something like this: 
![image](https://github.com/rdkit/rdkit/assets/42261966/ca90052e-c1fc-4d41-900a-963ba24739d9)

Is it possible to change the style of the arrow? For example, to have a double arrow, or change its head?

Thank you.

**Comment by greglandrum** [comment, marked as answer]
Hi @agrodet, this is not currently possible.

**Reply by agrodet** [reply]
Thank you very much for your prompt answer.
